pkg/aerospike/config.go

```diff
@@ -21,7 +21,9 @@ type AerospikeClientConfig struct {
 	// Contact point
 	host as.Host
 	// Config
-	genericConfig *AerospikeEndpointConfig
+	genericConfig              *AerospikeEndpointConfig
+	connectionQueueSize        int
+	openingConnectionThreshold int
```

Comment:
I think this should be part of the AerospikeEndpointConfig. AerospikeClientConfig is used for thing that change at runtime whereas the "connectionQueueSize" is only in the config. The naming I chose for the two config structs is not great 😓

Reply:
Done